compiler/src/dmd/traits.d

```diff
@@ -482,13 +482,19 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
     }
     if (e.ident == Id.isAbstractClass)
     {
-        return isTypeX(t => t.toBasetype().isTypeClass() &&
-                            t.toBasetype().isTypeClass().sym.isAbstract());
+        return isTypeX((t)
+        {
+            auto c = t.toBasetype().isTypeClass();
```

Comment:
these could be `const`

Reply:
Making `c` const fails as

```
compiler/src/dmd/traits.d(488,41): Error: mutable method `dmd.dclass.ClassDeclaration.isAbstract` is not callable using a `const` object
```

so skipping this.